internal/handlers/handlers.go

```diff
@@ -399,6 +405,15 @@ func ChannelsHandler(c *fiber.Ctx) error {
 	return c.JSON(apiResponse)
 }
 
+func containsString(arr []string, target string) bool {
```

Comment:
We already have a ContainsString function under pkg/utils package.

You can modify that or add your new function there if required.

I believe it's better for the function to be in pkg/utils than handlers